src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionTanInputViewController.swift

```diff
@@ -82,15 +90,14 @@ class ExposureSubmissionTanInputViewController: UIViewController, SpinnerInjecta
 
 extension ExposureSubmissionTanInputViewController {
 	enum Segue: String, SegueIdentifiers {
-		case sentSegue
 		case labResultsSegue
 	}
 }
 
 // MARK: - ExposureSubmissionNavigationControllerChild methods.
```

Comment:
Is this comment still relevant?

Reply:
changed to `ENANavigationControllerWithFooterChild` across all involved controllers.